ArmRobot/Assets/Scripts/Camera/RandomizerPositionCamera.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class RandomizerPositionCamera : MonoBehaviour
+{
+    public float changePositionCamera = 0.005f;
+    public float initialCameraPositionXAxis = 0.0f;
+    public float initialCameraPositionYAxis = 1.8f;
+    public float initialCameraPositionZAxis = 0.0f;
+    
+    public float rotationCamera = 1.0f;
+    public float initialCameraRotationXAxis = 90.0f;
+    public float initialCameraRotationYAxis = 0.0f;
+    public float initialCameraRotationZAxis = 0.0f;
+    
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        transform.position = new Vector3(initialCameraPositionXAxis, initialCameraPositionYAxis, initialCameraPositionZAxis);
+        transform.rotation = Quaternion.Euler(initialCameraRotationXAxis, initialCameraRotationYAxis, initialCameraRotationZAxis);
```

Comment:
Do this the other way around - set the variables in this file from the transform in Start() (as mentioned in the comments above).